.github/workflows/refresh-lockfiles.yml

```diff
@@ -13,13 +13,8 @@ name: Refresh Lockfiles
 
 
 on:
-  workflow_dispatch:
```

Comment:
This workflow will never be dispatched manually on the `SciTools/workflows`, rather that happens on the repo calling the reusable workflow.